cmd/guardian-cli/main.go

```diff
@@ -232,6 +272,141 @@ func main() {
 	}
 }
 
+func applyConfig(store *guardian.RedisConfStore, configFilePaths []string, logger logrus.FieldLogger) error {
+	for _, configFilePath := range configFilePaths {
+		var r io.Reader
+		if configFilePath == "-" {
+			r = os.Stdin
```

Comment:
Unfortunately, it looks like Kingpin gets confused by the dash thinking it's a flag. I think it might work to read from stdin if no configfilepaths are provided instead (e.g. `guardian-cli apply <<< 'my: config_data'`).

Reply:
Okay, this is working now.